lib/nimble_template/templates/variants/mix/template.ex

```diff
@@ -6,6 +6,10 @@ defmodule NimbleTemplate.Templates.Mix.Template do
   alias NimbleTemplate.Addons
   alias NimbleTemplate.Projects.Project
 
+  def pre_apply(%Project{} = project) do
+    install_addon_prompt(project, Addons.Mimic)
+  end
```

Comment:
```suggestion
  def pre_apply(%Project{} = project), do:
    install_addon_prompt(project, Addons.Mimic)
```

Reply:
Updated in [9782e2e](https://github.com/nimblehq/elixir-templates/pull/292/commits/9782e2ef5fef654f7a75a5bf77f6729d46406c43)